index.d.ts

```diff
@@ -0,0 +1,65 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import pino from 'pino';
+
+export = PinoHttp;
+
+declare function PinoHttp(opts?: PinoHttp.Options, stream?: pino.DestinationStream): PinoHttp.HttpLogger;
+declare function PinoHttp(stream?: pino.DestinationStream): PinoHttp.HttpLogger;
+
+declare namespace PinoHttp {
```

Comment:
AFAIR, namespaces are no longer a recommended TS pattern, using module is preferable.